forge-gui/res/cardsfolder/upcoming/harvestrite_host.txt

```diff
@@ -0,0 +1,10 @@
+Name:Harvestrite Host
+ManaCost:2 W
+Types:Creature Rabbit Citizen
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self,Rabbit.Other+YouCtrl | Execute$ TrigPump | TriggerDescription$ When CARDNAME or another Rabbit you control enters, target creature you control gets +1/+0 until end of turn. Then draw a card if this is the second time this ability has resolved this turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | NumAtt$ +1 | SubAbility$ DBDraw
+SVar:PlayMain1:TRUE
+SVar:DBDraw:DB$ Draw | ConditionCheckSVar$ Resolved | ConditionSVarCompare$ EQ2

```

Comment:
SVar doesn't exist

Reply:
I don't understand? It's referred to in TrigPump?

Reply:
the Condition one